modules/apps/object/object-service/src/main/java/com/liferay/object/model/impl/ObjectDefinitionImpl.java

```diff
@@ -286,6 +299,7 @@ public void setPreviousRESTContextPath(String previousRESTContextPath) {
 		_previousRESTContextPath = previousRESTContextPath;
 	}
 
+	private List<ObjectLayout> _defaultObjectLayouts;
```

Comment:
Can we have more than one default object layout?

Reply:
No, we can't have more than one default object layout. However, I used List<ObjectLayout> to check whether the variable is populated.